src/config/index.ts

```diff
@@ -231,6 +233,7 @@ export const RESUME_UPDATE_SUBMISSION_URL = UPDATE_AN_OVERSEAS_ENTITY_URL + TRAN
 
 // Update configs
 export const UPDATE_REVIEW_BENEFICIAL_OWNER_INDIVIDUAL_URL_WITH_PARAM_URL = UPDATE_REVIEW_BENEFICIAL_OWNER_INDIVIDUAL_URL + REVIEW_BENEFICIAL_OWNER_INDEX_PARAM + ID;
+export const UPDATE_REVIEW_BENEFICIAL_OWNER_GOV_URL_WITH_PARAM_URL = UPDATE_REVIEW_BENEFICIAL_OWNER_GOV_URL + REVIEW_BENEFICIAL_OWNER_INDEX_PARAM + ID;
```

Comment:
not sure if we need the `+ ID` added at the end of the config and routes. Since you've gone with using index query param and only ever pass 1 number. Can't see it used anywhere and I've removed it in my ticket (from BO individual and BO other) and testing it it all seems to still work as expected.

Reply:
ignore this, think this was late on Friday and don't trust that I was thinking straight by that time :D